.github/review-bot.yml

```
@@ -9,24 +9,20 @@ rules:
   - name: Relay and system files
     condition:
       include:
-        - ^relay\/kusama\/.*
-        - ^relay\/polkadot\/.*
+        - ^relay\/.*
         - ^system-parachains\/.*
-        - ^CHANGELOG$
-      exclude: 
-        - ^relay\/.+\.adoc$
+        - ^CHANGELOG.md$
     type: fellows
     minRank: 3
     minApprovals: 4
   - name: General Files
     condition:
       include:
         - '.*'
-      exclude: 
-        - ^relay\/kusama\/.*
-        - ^relay\/polkadot\/.*
-        - ^\.github/.*
+      exclude:
+        - ^relay\/.*
         - ^system-parachains\/.*
-        - ^target\/.*
+        - ^\.github/.*
+        - ^CHANGELOG.md$
```

Also here. Btw, by excluding it here it won't be protected by the review bot as now it doesn't fit into any `include` rules.

```suggestion
        - ^CHANGELOG$
```

---

It is part of the include rules for "Relay and system files"?

---

Ah yes, you are right.

I got it confused with the way git showed the file changes and missread the `exclude` section.